ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -20,7 +20,7 @@ let prototyped_of_field = function
   | "host", "last_software_update" ->
       Some "22.20.0"
   | "pool", "coordinator_bias" ->
-      Some "22.34.0-next"
```

Comment:
Please split this change into a separate commit. Having separate changes in different commits makes it easier to reverse any of them separately if needed